### PR TITLE
ui: restyle markets page

### PR DESCRIPTION
**Pull request by buck54321**
The markets page is restyled. The new style is responsive. There are also some other stylistic changes that apply across the app. Still need to do a little housekeeping and add some docs.

**desktop**

**smaller**

**mobile**

### CODE REVIEW

**Review by JoeGruffins — APPROVED** [review]
Wow. I can finally trade on my phone. Looks amazing.

File: client/webserver/site/src/css/main.scss [hunk 1]
```diff
@@ -60,39 +60,44 @@ input:invalid {
   overflow-y: auto;
 }
 
-.overflow-x-hidden {
+.overflow-y-hidden {
+  overflow-y: hidden;
+}
+
+.overflow-x-hidden,
+.stylish-overflow.overflow-x-hidden {
```
Comment (chappjc) [inline client/webserver/site/src/css/main.scss:68]: Aren't these redundant?

```html
<element class="overflow-x-hidden">
```

and

```html
<element class="stylish-overflow overflow-x-hidden">
```

**Review by chappjc — COMMENTED** [review]
I think we should get https://github.com/decred/dcrdex/pull/1663 in first if you don't mind.

This PR is probably the best time to design for a (much much smaller) recent matches table, perhaps even hideable, that looks like the ones on trex and bn that I linked to on that PR.

**Comment by buck54321** [comment]
I have a little work left to do on the recent matches section.

File: client/webserver/site/src/js/locales.ts [hunk 2]
```diff
@@ -205,7 +217,9 @@ export const zhCN: Locale = {
   [ID_EPOCH]: '时间',
   [ID_API_ERROR]: '接口错误',
   [ID_ADD]: '加',
-  [ID_CREATE]: '创建'
+  [ID_CREATE]: '创建',
+  [ID_AVAILABLE]: '可用',
+  [ID_IMMATURE]: '不成熟'
```
[the remark this answers is not in the document]
Comment (buck54321) [reply client/webserver/site/src/js/locales.ts:230]: These used to be from the Go template e.g. `[[[available]]]`, but now all balance rows are generated here, so I just copied them.

**Review by chappjc — APPROVED** [review]
Looking great!  Just a silly conflict with a candles change to resolve.

**Pull request merged**